app/ui/src/app/integration/import-export/import/integration-import.component.scss

```diff
@@ -0,0 +1,25 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow: visible;
+}
+
+syndesis-list-toolbar {
+  flex: 0 0 auto;
+}
+
+syndesis-loading {
+  flex: 1 1 auto;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  // Dock the scrollbar to the right
+  margin-right: -20px;
+  padding-right: 20px;
+}
+
+.max-height {
+  height:100%
+}
```

Comment:
This chunk of SASS basically duplicates to the point what we already had at `app/integration/list-page/list-page.component.scss`. This kind of code duplication must be avoided by all means, since it only increases the overall footprint of the application bundle and prevents CSS reusability and maintainability. You have several workarounds for this:

- Move this chunk of code to a shared location and import it from both components (remember that the `styleUrls` property expects an array with not one but many references).
- Turn this piece of code into a SASS partial and import it from each component's own stylesheet, should you feel that each component will eventually feature its own CSS nuances.
- Move parts of the code to SASS `%` extension classnames and use them internally within these selectors.
- Use a global mixin and import it directly.

In addition to the approaches above, you should strive to stick to the app variables we defined at `/scss/base/_variables.scss` in order to keep consistency in the the UI's look and feel. So whenever you're dealing with component CSS variables, you should strive to avoid hardcoded values as much as you can.